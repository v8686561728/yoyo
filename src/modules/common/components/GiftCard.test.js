import React from "react";
import { shallow } from "../../../enzyme";
import { shallowToJson } from "enzyme-to-json";
import { GiftCard } from "./GiftCard";

let props = {
  classes: {
    card: "card"
  },
  giftCard: {
    cardName: "cardName"
  }
};
it("should render GiftCard", () => {
  let wrapper = shallow(<GiftCard {...props} />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});