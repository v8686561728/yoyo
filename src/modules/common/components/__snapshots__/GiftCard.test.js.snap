// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render GiftCard 1`] = `
<WithStyles(ForwardRef(Card))
  className="card"
>
  <WithStyles(ForwardRef(CardHeader))
    action={null}
    avatar={
      <ForwardRef(WithStyles)
        aria-label="Gift Card"
      >
        c
      </ForwardRef(WithStyles)>
    }
    subheader="undefined points"
    title={
      <Link
        replace={false}
        style={
          Object {
            "textDecoration": "none ",
          }
        }
        to="/giftCards/undefined"
      >
        cardName
      </Link>
    }
  />
  <Link
    replace={false}
    style={
      Object {
        "textDecoration": "none ",
      }
    }
    to="/giftCards/undefined"
  >
    <WithStyles(ForwardRef(CardMedia))
      title="Hurry! Only undefined remaining"
    />
  </Link>
  <WithStyles(ForwardRef(CardContent))>
    <WithStyles(ForwardRef(Typography))
      component="p"
    />
  </WithStyles(ForwardRef(CardContent))>
</WithStyles(ForwardRef(Card))>
`;
